sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -4,13 +4,11 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  #commit b38855ed815ffd076bfde5e5e06170ca4f723dc1
-  #Merge: e6343f4 6a2c548
-  #Author: Piotr Balcer <piotr.balcer@intel.com>
-  #Date:   Thu Oct 5 12:15:42 2023 +0200
-  #    Merge pull request #920 from jsji/localcopy
-  #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  # commit 3653e582e00737302ba0469a978aadc16448b3ea
+  # Author: Fabio <fabio.mestre@codeplay.com>
+  # Date:   Wed Oct 18 16:25:29 2023 +0100
+  #    [OpenCL] Allow upstream to manually set the OpenCL target (#969)
+  set(UNIFIED_RUNTIME_TAG 3653e582e00737302ba0469a978aadc16448b3ea)
```

Comment:
Once https://github.com/intel/llvm/pull/11344 merges this will conflict but we should take the commit from that PR as they should include the OpenCL adapter code already.